src/rpc/quorums.cpp

```diff
@@ -246,7 +246,27 @@ static RPCHelpMan quorum_info()
         RPCResult{
             RPCResult::Type::OBJ, "", "",
             {
-                {RPCResult::Type::ELISION, "", ""}
+                {RPCResult::Type::NUM, "height", "Quorum Height"},
+                {RPCResult::Type::STR, "type", "Quorum type"},
+                {RPCResult::Type::STR_HEX, "quorumHash", "Quorum hash"},
+                {RPCResult::Type::NUM, "quorumIndex", "Quorum index (applicable only to rotated quorums)."},
+                {RPCResult::Type::STR_HEX, "minedBlock", "Blockhash where the commitment was mined."},
+                {RPCResult::Type::NUM, "previousConsecutiveDKGFailures", "Number of previous consecutive DKG failures."},
+                {RPCResult::Type::ARR, "members", "Members of quorum",
+                    {
+                        {RPCResult::Type::OBJ, "", "",
+                        {
+                            {RPCResult::Type::STR_HEX, "proTxHash", "ProTx of collaterial"},
+                            {RPCResult::Type::STR, "service", "Address of node. Legacy, will be deprecated in further versions. See `addresses` instead."},
+                            {RPCResult::Type::ARR, "addresses", "Addresses of node",
+                                // TODO - expand it when extended addresses are fully implemented
+                                {{RPCResult::Type::ELISION, "", ""}},
+                            },
+                            {RPCResult::Type::STR_HEX, "pubKeyOperator", "The operator BLS public key"},
```

Comment:
We've already defined `pubKeyOperator` (see below), please use `GetRpcResult()`

https://github.com/dashpay/dash/blob/f650f655879f951537000ab94f7cca40752bec93/src/evo/core_write.cpp#L60